### ISSUE TITLE
Spell: Flurry

### ISSUE THREAD

**Issue by Tyloris** [WC5E/Warcraft-5e-Conversion#810]
We've had a bit of conversation about this spell in the spell-updates channel, so I'm posting this here. Two main points on this cantrip:
- First, it could easily be a shaman spell instead of a mage-only one. The current format doesn't quite match what a shaman would cast, however.
- Second, the existing cantrip is kind of strong, a little too much so. There aren't really any cantrips that can hit more than 1-2 people at a time, and even then only if they are standing next to each other (acid splash, green flame blade) or the caster (sword burst, word of radiance). The only cantrip that does is eldritch blast and it doesn't get the ability to do so until 11th level. On top of that, the cantrip does 3d4 damage to a single target, making it the highest damaging cantrip out there. The closest are poison spray and toll the dead at 1d12. 

In short, the spell should either be cut or changed.

What I'm envisioning the spell as is a blast of cold air and ice/snow, basically a cantrip level version of ice storm and sleet storm. The cantrip needs to avoid overlap with other cantrips, particularly ray of frost and frostbite. One option is [this spell from DnDBeyond.](https://www.dndbeyond.com/spells/122590-chill-wind) It pushes creatures back, which is a useful method of battlefield control, especially in combination with lightning lure. An alternative option is making it deal cold damage in a radius, such as dealing 1dX damage to one creature and a lesser amount (spellcasting modifier?) to any adjacent creatures, or by dealing 1d4 cold damage to all creatures in a 5 foot radius.
